src/services/item/service.ts

```diff
@@ -454,6 +461,7 @@ export class ItemService {
     }
 
     const descendants = await itemRepository.getDescendants(item);
+    item.name = name || item.name;
```

Comment:
We could make the renaming a bit more explicit by using an if statement:
```suggestion
    if (args.name) {
      // assign a new name to the copied item
      // this supports the duplicate item feature
      item.name = args.name
    }
```